modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/FieldsTable.js

```diff
@@ -65,11 +66,33 @@ function FieldsTable({portletNamespace}) {
 			useTemplateMappingRef.current = false;
 		};
 
+		const handleExportFileFormatUpdated = ({
+			selectedExportFileFormat,
+			selectedSchema,
+		}) => {
+			if (
+				selectedExportFileFormat === 'CSV' &&
+				selectedSchema ===
+					'com.liferay.object.admin.rest.dto.v1_0.ObjectDefinition'
```

Comment:
`com.liferay.object.admin.rest.dto.v1_0.ObjectDefinition` is hardcoded 3 times. I think it would be better to centralize it.

Reply:
Agree with this